tools/test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -ex
+
+TESTS=$1
+CONTAINER=$2
+
+[ -z "$CONTAINER" ] && CONTAINER='ghcr.io/dracut-ng/fedora'
+
+# if podman is available prefer podman over docker
+[ -x "$(command -v podman)" ] && alias docker=podman
+
+docker run --rm -it --device=/dev/kvm -v "$PWD"/:/z "$CONTAINER" /z/tools/test-github.sh id "$TESTS"
```

Comment:
I would prefer we only officially support Podman rather than both. There are subtle differences between the two and I don't want us to make people think we are capable of dealing with issues from Docker.